app/scripts/migrations/00005_change_job_dialogs_column_type/index.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE jobs 
```

Comment:
Would be good to get a comment for each of these migrations explaining why they exist. Like what is the column type of dialogs currently and why does that not suffice?